ci_test/common_python/tools.py

```diff
@@ -1019,7 +1019,10 @@ def gpus_per_node(lbann):
 
 
 # Get the environment variables for Distconv.
-def get_distconv_environment():
+def get_distconv_environment(init_nvshmem=False):
     # TODO: Use the default halo exchange and shuffle method. See https://github.com/LLNL/lbann/issues/1659
     return {"LBANN_DISTCONV_HALO_EXCHANGE": "AL",
-            "LBANN_DISTCONV_TENSOR_SHUFFLER": "AL"}
+            "LBANN_DISTCONV_TENSOR_SHUFFLER": "AL",
+            "LBANN_KEEP_ERROR_SIGNALS": "1",
+            "LBANN_INIT_NVSHMEM": (1 if init_nvshmem else 0),
```

Comment:
I might actually suggest just always setting this to 1. If NVSHMEM is enabled, it should be `1`. If NVSHMEM is not enabled, its value will be ignored. You can then get rid of all the `init_nvshmem` cluttter.

Reply:
Yeah, `LBANN_INIT_NVSHMEM` is ignored if `LBANN_HAS_NVSHMEM` is not set.

Reply:
@benson31 Here is a fix for that.